packages/server/src/lib/email.js

```diff
@@ -70,6 +74,8 @@ async function deliverEmail({
             `service=${process.env.DD_SERVICE}`,
             `env=${process.env.DD_ENV}`,
             `version=${process.env.DD_VERSION}`,
+            `dd_trace_id=${traceId}`,
+            `dd_span_id=${spanId}`,
```

Comment:
Per the above suggestion:
```suggestion
            ...ddTraceTags,
```